Decompiler/Options.cs

```diff
@@ -42,6 +42,18 @@ class Options
             HelpText = "Verify checksums and signatures.")]
         public bool VerifyVPKChecksums { get; set; }
 
+        [Option('d', "vpk_dump",
```

Comment:
I think this should be the other way around, like 'vpk_decompile', so that by default it only dumps the file.